admin/includes/router.php

```diff
@@ -37,12 +38,16 @@ public function execute()
     }
 
     Routes::setRoute('home', function() {
-      HomeController::CreateView('Home');
```

Comment:
Add condition if the admin is logged in then create the view of Home else log in,